saleor/account/i18n.py

```diff
@@ -38,6 +38,10 @@
     "zip": "ZIP code",
 }
 
+CountryAreaValues = namedtuple("CountryAreaValues", ["value", "choices"])
+
+COUNTRY_AREA_MAP = {"CH": CountryAreaValues("Graubünden", ["Graubünden"])}
```

Comment:
I may not fully understand the requirements, but do we really need to have and maintain such a map on the core side? It seems like we would have to cover a large number of countries and regions, and manually paste them into the Saleor code. Additionally, we would have to update the list of choices whenever someone reports a missing value, and create a new release with an extended map that would need to be ported to all Saleor versions greater than the requested one.